docs/content/integrations/dbt/reference.mdx

```diff
@@ -485,6 +486,46 @@ Note that Dagster allows the optional specification of a [`code_version`](/conce
 
 ---
 
+## Loading dbt tests as asset checks <Experimental />
+
+<Note>
+  <strong>Dagster asset checks are currently an experimental feature. </strong>{" "}
+  To provide feedback, join our{" "}
+  <a href="https://github.com/dagster-io/dagster/discussions/16527">
+    GitHub discussion
+  </a>{" "}
+  to share your use case with Dagster asset checks and dbt.
+</Note>
+
+Dagster allows you to model your existing dbt tests as [asset checks](/concepts/assets/asset-checks) that assert data quality properties.
```

Comment:
Nitpick: the "that assert data quality properties at the end of this sentence" makes it seem like there's some other common kind of asset check. It could be clearer to leave this out. I'll leave this decision up to you though.